### ISSUE TITLE
Allow paging instead of infinite scroll in feeds

### ISSUE THREAD

**Issue by oscarnylander** [dessalines/jerboa#411]
Infinite scroll can unintentionally promote unhealthy consumption patterns. Paging can help create natural 'exit points', which in turn improves user agency over their consumption patterns.

**Comment by dessalines** [comment]
I agree with this fully. If anyone wants to work on that, I'd be glad to merge it.

**Comment by Die4Ever** [comment]
This would also be a good workaround for #1003
